clients/iOS/Grapevine/Grapevine/CalendarViewController.swift

```diff
@@ -40,30 +41,25 @@ class CalendarViewController: UIViewController, CVCalendarViewDelegate, CVCalend
     }
     
     
-    
-    @IBAction func todayPressed(){
-        
-    }
-    
-    @IBAction func weekMonthPressed(sender: UIBarButtonItem){
-        let monthStr = "Month"
-        let weekStr = "Week"
-        if sender.title == weekStr {
-            calendarView.calendarMode = .WeekView
-            sender.title = monthStr
+    @IBAction func changeView(sender: UIBarButtonItem){
+        if self.inMonthView {
+            self.toWeekView()
+            sender.title = "Month"
```

Comment:
Nah I'm going into Week Mode but then I change the title of the button to read Month so the button can be clicked while in Week Mode to go to Month mode